initfiles/bash/etc/init.d/hpcc_common.in

```diff
@@ -505,6 +505,8 @@ startCmd() {
       SENTINEL_CHECK=0
     fi
 
+    # sleep is necessary to insure the eval $startcmd actually kicked off
+    sleep 1
```

Comment:
The issue I was running into, is where the start-stop-daemon call would be launched, but we would get into the check_status wait loop before the actual init script had time to set up it's pid file.  With my changes to pid.sh to look for a process with no init_pid file, this meant that we would exit the loop before the start command had setup it's pid file.  

Thinking on this though now that I've had a day away from the problem... I could instead of using the pid file that we create in the actual component init scripts, try to instead use the $compName_init.pid file thatt is created directly with the start-stop-daemon call.